### DISCUSSION TITLE
Google Analytics: Ability to filter between GA UA and GA4

### DISCUSSION THREAD

**Discussion by jpool-cf** [MisterPhilip/omnibug#225] [Ideas]
Now that GA4 has taken over and GA UA has essentially reached end of life for customers not on the paid 360 version of Google Analytics, it would be great to be able to filter out GA UA hits while debugging

Maybe just separating out the current filter into 2 distinct options?

Thanks!

**Comment by MisterPhilip** [comment]
Yeah, this is something I've been toying around with. I previously had it broken out between ga.js and analytics.js, but most people didn't seem to understand/care about the difference so they were merged together. GA4 is a different beast compared UA, so I agree that it is worthwhile to split the two.

I am working on a new version of Omnibug which already had this split out; I'm not sure if I will backport it to get it out sooner, but it is coming!

**Reply by jpool-cf** [reply]
Awesome! Looking forward to it

**Reply by MisterPhilip** [reply]
@jpool-cf - this is going live with [v1.21.0](https://github.com/MisterPhilip/omnibug/discussions/229)

**Reply by jpool-cf** [reply]
Amazing, thank you!
